app/src/main/res/xml/grayscale_icon_map.xml

```diff
@@ -458,6 +460,7 @@
     <icon drawable="@drawable/grab" package="com.grabtaxi.passenger" name="Grab" />
     <icon drawable="@drawable/grapheneos_apps" package="app.grapheneos.apps" name="Apps" />
     <icon drawable="@drawable/grapheneos_auditor" package="app.attestation.auditor" name="Auditor" />
+    <icon drawable="@drawable/grapheneos_pdf" package="app.grapheneos.pdfviewer" name="PDF Viewer" />
```

Comment:
The line 463 is from official GitHub repository of GrapheneOS, and the line 464 is from Google Play. The original application name is PDF Viewer, no "Secure"
![image](https://user-images.githubusercontent.com/40175769/201282001-3562a35a-9c44-4a8c-9b91-3f62bb96446c.png)